src/lib/stores/contractStore.js

```diff
@@ -0,0 +1,85 @@
+import { persisted } from 'svelte-local-storage-store'
+import { v4 as uuidv4 } from 'uuid'
+import { Contract, StrKey } from '@stellar/stellar-sdk'
+import { error } from '@sveltejs/kit'
+import { get } from 'svelte/store'
+
+/**
+ * @typedef {Object} SavedContract
+ * @property {string} id Unique identifier for this contract
+ * @property {string} contractId The Stellar contract ID
+ * @property {string} name Human-readable name for this contract
+ * @property {string} [description] Optional description of the contract
+ */
+
+function createContractStore() {
+    /** @type {import('svelte/store').Writable<{savedContracts: SavedContract[], currentContract: SavedContract|null}>} */
```

Comment:
Let's explore making a `@typedef` for the `contractStore`, since it's a little unclear what's happening when it's defined within the `Writable<...>` part.